CHANGELOG.md

```diff
@@ -7,13 +7,48 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [0.1.47] - 2021-12-15
 ### Changes
-- Add human-readable changelog for 0.1.45 and 0.1.46
-- Fix bugs where kubeletconfig gets deletes when unapplying
-- Remove regex checks for url-encoded content
-- ScanSetting: Introduce special `@all` role to match all nodes
-- Add documentation for E2E test
-- Inherit scheduling for workloads from Compliance Operator manager
-- Add status descriptor for CRDs
+
+ - enhancements
+      - Add status descriptor for CRDs - the CRDs' status attributes were annotated
+        with [status descriptors](https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md)
+        that would provide a nicer UI for the status subresources of Compliance Operator's
+        CRs in the OpenShift web UI console.
+      - ScanSetting: Introduce special `@all` role to match all nodes - Introduces
+        the ability to detect `@all` in the ScanSettings roles. When used, the
+        scansettingbinding controller will set an empty `nodeSelector` which
+        will then match all nodes available. This is mostly intended for
+        non-openshift deployments
```

Comment:
nit: add punctuation to the end of this note to be consistent with all other lines.

Reply:
done, thanks.